### PR TITLE
NXP-29229 explorer templates for deployment

### PR DESCRIPTION
**Pull request by atchertchian**

### CODE REVIEW

**Comment by atchertchian** [comment]
forgot to update ftests checking readmes -> will rebase and force push with fixes

File: ReadMe.md [hunk 1]
```diff
@@ -26,10 +27,16 @@ The following documentation items are also extracted:
...
- - `org.nuxeo.apidoc.hide.seam.components`: Hide Seam components introspection (moved to nuxeo-apidoc-jsf since 11.1)
+## Configuration
+
+The template `explorer-sitemode` enables the nuxeo.conf property `org.nuxeo.apidoc.site.mode` and
+defines an anonymous user.
+The property `org.nuxeo.apidoc.site.mode` comes with a more user friendly design and hides the current
+"live" distribution from display and API.
+
+The template `explorer-virtualadmin` disables the usual `Administrator` user creation at database
+initialization and add a virtual admin user with name `apidocAdmin`, which password can be changed using
```
Comment (nelsonsilva) [inline ReadMe.md:38]:
```suggestion
initialization and adds a virtual admin user with name `apidocAdmin`, whose password can be changed using
```

File: ftests/explorer-ftests-base/src/main/resources/data/apidoc_readme.txt [hunk 2]
```diff
@@ -26,10 +27,16 @@ The following documentation items are also extracted:
...
- - `org.nuxeo.apidoc.hide.seam.components`: Hide Seam components introspection (moved to nuxeo-apidoc-jsf since 11.1)
+## Configuration
+
+The template `explorer-sitemode` enables the nuxeo.conf property `org.nuxeo.apidoc.site.mode` and
+defines an anonymous user.
+The property `org.nuxeo.apidoc.site.mode` comes with a more user friendly design and hides the current
+"live" distribution from display and API.
+
+The template `explorer-virtualadmin` disables the usual `Administrator` user creation at database
+initialization and add a virtual admin user with name `apidocAdmin`, which password can be changed using
```
Comment (nelsonsilva) [inline ftests/explorer-ftests-base/src/main/resources/data/apidoc_readme.txt:38]:
```suggestion
initialization and adds a virtual admin user with name `apidocAdmin`, whose password can be changed using
```

File: ftests/explorer-ftests-base/src/main/resources/data/sample_export/apidoc/Bundles/bg-apidoc/bundle-org-nuxeo-apidoc-core/35dfe998.blob [hunk 3]
```diff
@@ -26,10 +27,16 @@ The following documentation items are also extracted:
...
- - `org.nuxeo.apidoc.hide.seam.components`: Hide Seam components introspection (moved to nuxeo-apidoc-jsf since 11.1)
+## Configuration
+
+The template `explorer-sitemode` enables the nuxeo.conf property `org.nuxeo.apidoc.site.mode` and
+defines an anonymous user.
+The property `org.nuxeo.apidoc.site.mode` comes with a more user friendly design and hides the current
+"live" distribution from display and API.
+
+The template `explorer-virtualadmin` disables the usual `Administrator` user creation at database
+initialization and add a virtual admin user with name `apidocAdmin`, which password can be changed using
```
Comment (nelsonsilva) [inline ftests/explorer-ftests-base/src/main/resources/data/sample_export/apidoc/Bundles/bg-apidoc/bundle-org-nuxeo-apidoc-core/35dfe998.blob:38]:
```suggestion
initialization and adds a virtual admin user with name `apidocAdmin`, whose password can be changed using
```

File: ftests/explorer-ftests-base/src/main/resources/data/sample_export/apidoc/Bundles/bg-apidoc/bundle-org-nuxeo-apidoc-repo/a419a275.blob [hunk 4]
```diff
@@ -26,10 +27,16 @@ The following documentation items are also extracted:
...
- - `org.nuxeo.apidoc.hide.seam.components`: Hide Seam components introspection (moved to nuxeo-apidoc-jsf since 11.1)
+## Configuration
+
+The template `explorer-sitemode` enables the nuxeo.conf property `org.nuxeo.apidoc.site.mode` and
+defines an anonymous user.
+The property `org.nuxeo.apidoc.site.mode` comes with a more user friendly design and hides the current
+"live" distribution from display and API.
+
+The template `explorer-virtualadmin` disables the usual `Administrator` user creation at database
+initialization and add a virtual admin user with name `apidocAdmin`, which password can be changed using
```
Comment (nelsonsilva) [inline ftests/explorer-ftests-base/src/main/resources/data/sample_export/apidoc/Bundles/bg-apidoc/bundle-org-nuxeo-apidoc-repo/a419a275.blob:38]:
```suggestion
initialization and adds a virtual admin user with name `apidocAdmin`, whose password can be changed using
```

File: ftests/explorer-ftests-base/src/main/resources/data/sample_export/apidoc/Bundles/bg-apidoc/bundle-org-nuxeo-apidoc-webengine/f6e1ec6e.blob [hunk 5]
```diff
@@ -26,10 +27,16 @@ The following documentation items are also extracted:
...
- - `org.nuxeo.apidoc.hide.seam.components`: Hide Seam components introspection (moved to nuxeo-apidoc-jsf since 11.1)
+## Configuration
+
+The template `explorer-sitemode` enables the nuxeo.conf property `org.nuxeo.apidoc.site.mode` and
+defines an anonymous user.
+The property `org.nuxeo.apidoc.site.mode` comes with a more user friendly design and hides the current
+"live" distribution from display and API.
+
+The template `explorer-virtualadmin` disables the usual `Administrator` user creation at database
+initialization and add a virtual admin user with name `apidocAdmin`, which password can be changed using
```
Comment (nelsonsilva) [inline ftests/explorer-ftests-base/src/main/resources/data/sample_export/apidoc/Bundles/bg-apidoc/bundle-org-nuxeo-apidoc-webengine/f6e1ec6e.blob:38]:
```suggestion
initialization and adds a virtual admin user with name `apidocAdmin`, whose password can be changed using
```

File: modules/nuxeo-apidoc-repo/src/test/resources/apidoc_snapshot/apidoc_readme.txt [hunk 6]
```diff
@@ -26,10 +27,16 @@ The following documentation items are also extracted:
...
- - `org.nuxeo.apidoc.hide.seam.components`: Hide Seam components introspection (moved to nuxeo-apidoc-jsf since 11.1)
+## Configuration
+
+The template `explorer-sitemode` enables the nuxeo.conf property `org.nuxeo.apidoc.site.mode` and
+defines an anonymous user.
+The property `org.nuxeo.apidoc.site.mode` comes with a more user friendly design and hides the current
+"live" distribution from display and API.
+
+The template `explorer-virtualadmin` disables the usual `Administrator` user creation at database
+initialization and add a virtual admin user with name `apidocAdmin`, which password can be changed using
```
Comment (nelsonsilva) [inline modules/nuxeo-apidoc-repo/src/test/resources/apidoc_snapshot/apidoc_readme.txt:38]:
```suggestion
initialization and adds a virtual admin user with name `apidocAdmin`, whose password can be changed using
```

**Pull request merged**